src/components/select/demoSelectHeader/style.css

```diff
@@ -0,0 +1,19 @@
+.demo-header-searchbox {
+  border: none;
+  outline: none;
+  height: 100%;
+  width: 100%;
+  padding: 0;
+}
+
+.demo-select-header {
+  box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.1), 0 0 0 0 rgba(0, 0, 0,
+  0.14), 0 0 0 0 rgba(0, 0, 0, 0.12);
+  padding-left:  40px;
+  height: 48px;
+  cursor: pointer;
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: auto;
+}
```

Comment:
Hmmm, the build system change sounds great, but a lot of work for the material team so for the sake of everyone, let's table that.

I discovered that I can use a `data-` attribute to add a class to the `menu-container` element. So I think I'll just go that route. Good call adding that hook!